services/snapshot/client.go

```diff
@@ -130,14 +167,20 @@ func toKind(kind snapshotapi.Kind) snapshot.Kind {
 	if kind == snapshotapi.KindActive {
 		return snapshot.KindActive
 	}
+	if kind == snapshotapi.KindView {
+		return snapshot.KindView
+	}
 	return snapshot.KindCommitted
 }
 
 func toInfo(info snapshotapi.Info) snapshot.Info {
 	return snapshot.Info{
-		Name:   info.Name,
-		Parent: info.Parent,
-		Kind:   toKind(info.Kind),
```

Comment:
The implementation of `toKind` doesn't look correct after the addition of the `view` type.